### PR TITLE
Cleaec/tdi 47523 jdk11

### PR DESCRIPTION
**Pull request by clesaec**
https://jira.talendforge.org/browse/TDI-47523 : jdk 11 for connectors-se

### CODE REVIEW

**Review by dmytrogrygorenko — DISMISSED** [review]

File: Jenkinsfile [hunk 1]
```diff
@@ -244,7 +245,7 @@ spec:
                         }
                     }
                     environment {
-                        LIST_FILE= sh(returnStdout: true, script: "find \$(pwd) -type f -name 'jacoco.xml'  | sed 's/.*/&/' | tr '\n' ','").trim()
+                        LIST_FILE= sh(returnStdout: true, script: "for rep in `find \$(pwd) -type d -name \"jacoco-aggregate\" | grep \"/target/\"`; do ls \${rep}/jacoco.xml; done  | sed 's/.*/&/' | tr '\n' ','").trim()
```
Comment (fdesiles) [inline Jenkinsfile:248]: Hello @clesaec , could you please explain this modification ? Was it required in the scope of related ticket if not, why changing the way the files are listed ?
Thanks

File: jdbc/pom.xml [hunk 2]
```diff
@@ -240,7 +240,7 @@
                 </repository>
                 <repository>
                     <id>redshift</id>
-                    <url>http://s3.amazonaws.com/redshift-maven-repository/release</url>
+                    <url>ttps://mvnrepository.com/artifact/</url>
```
Comment (fdesiles) [inline jdbc/pom.xml:243]: Hello @clesaec , could you please explain why this repo has been modified ? We currently have some topics on Redshift,  I'd prefer not touch related content until we are all clear.

File: Jenkinsfile (hunk 1 above)
Comment (clesaec) [reply Jenkinsfile:248]: jacoco.xml file regroup file for unit test only and file that regroup unit test and it test (those that are in folder "jacoco-aggregate"). So, it reduce the number of file to analyse on sonar side.

File: jdbc/pom.xml (hunk 2 above)
Comment (clesaec) [reply jdbc/pom.xml:243]: was an issue on CI, yves give me this trick to fix it (otherwise, the CI is not OK)

File: Jenkinsfile (hunk 1 above)
Comment (fdesiles) [reply Jenkinsfile:248]: Ok thanks !

File: jdbc/pom.xml (hunk 2 above)
Comment (fdesiles) [reply jdbc/pom.xml:243]: Noted, thanks.

**Review by fdesiles — DISMISSED** [review]
Rebase LGTM

**Review by dmytrogrygorenko — APPROVED** [review]
LGTM

**Review by ypiel-talend — APPROVED** [review]
LGTM

**Pull request merged**